### PR TITLE
Update gems

### PR DESCRIPTION
**Pull request by SuperTux88**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -32,49 +32,50 @@ gem "simple_captcha2", "0.4.3", require: "simple_captcha"
...
 
-gem "activerecord-import", "0.17.1"
+gem "activerecord-import", "0.19.1"
 
 # File uploading
 
-gem "fog-aws",     "1.2.1"
-gem "carrierwave", "0.11.2"
-gem "mini_magick", "4.6.1"
+gem "carrierwave", "1.1.0"
```
Comment (diaspora-code-review) [inline Gemfile]: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `carrierwave` should appear before `fog-aws`.

File: Gemfile [hunk 2]
```diff
@@ -274,24 +275,24 @@ end
...
   gem "cucumber-api-steps", "0.13", require: false
 
   # General helpers
 
   gem "factory_girl_rails", "4.8.0"
-  gem "timecop",            "0.8.1"
-  gem "webmock",            "2.3.2", require: false
-  gem "shoulda-matchers",   "3.1.1"
+  gem "shoulda-matchers",   "3.1.2"
+  gem "timecop",            "0.9.1"
```
Comment (diaspora-code-review) [inline Gemfile]: Gems should be sorted in an alphabetical order within their section of the Gemfile. Gem `shoulda-matchers` should appear before `webmock`.

File: Gemfile (hunk 1 above)
Comment (SuperTux88) [inline Gemfile]: ¯\_(ツ)_/¯

**Review by skyschub — APPROVED** [review]
Travis is happy, Pronto is drunk, everything looks fine.

**Pull request merged**